docs/usage.md

```diff
@@ -162,87 +233,268 @@ Note that you can use the same configuration setup to save sets of reference fil
 
 ## AWS Batch specific parameters
-Running the pipeline on AWS Batch requires a couple of specific parameters to be set according to your AWS Batch configuration. Please use the `-awsbatch` profile and then specify all of the following parameters.
+
+Running the pipeline on AWS Batch requires a couple of specific parameters to be set according to your AWS Batch configuration. Please use [`-profile awsbatch`](https://github.com/nf-core/configs/blob/master/conf/awsbatch.config) and then specify all of the following parameters.
+
 ### `--awsqueue`
+
 The JobQueue that you intend to use on AWS Batch.
+
 ### `--awsregion`
-The AWS region to run your job in. Default is set to `eu-west-1` but can be adjusted to your needs.
+
+The AWS region in which to run your job. Default is set to `eu-west-1` but can be adjusted to your needs.
+
+### `--awscli`
+
+The [AWS CLI](https://www.nextflow.io/docs/latest/awscloud.html#aws-cli-installation) path in your custom AMI. Default: `/home/ec2-user/miniconda/bin/aws`.
 
 Please make sure to also set the `-w/--work-dir` and `--outdir` parameters to a S3 storage bucket of your choice - you'll get an error message notifying you if you didn't.
 
-## Other command line parameters
+## Processing parameters
+
+### `--trim5`
+
+Integer indicating the number of basepairs to trim from the 5' end of the reads.
+
+```bash
+--trim5 '[number of  bp to trim from 5prime end]'
+```
+
+### `--polyA`
+
+Integer indicating the maximum number of As at the 3' end of a read before considering them as poly-As and trimming them.
+
+```bash
+--polyA '[Maximum 3prime end read As]'
+```
+
+### `--multimappers`
+
+Boolean flag activating the [multimapper recovery strategy](https://t-neumann.github.io/slamdunk/docs.html#multimapper-reconciliation). Will either use the bed file supplied by `--mapping` or alternatively use the plain `--bed` file.
+
+### `--quantseq`
+
+Boolean flag deactivating the conversion-aware scoring scheme in [NextGenMap](http://cibiv.github.io/NextGenMap/). This will result in always zero T>C reads and render [DESeq2](https://doi.org/10.1186/s13059-014-0550-8) analysis meaningless.
 
-<!-- TODO nf-core: Describe any other command line flags here -->
+### `--endtoend`
+
+Boolean flag to activate end-to-end mapping in [NextGenMap](http://cibiv.github.io/NextGenMap/).
+
+### `--min_coverage`
+
+Minimum coverage to call a SNP as integer.
+
+```bash
+--min_coverage '[Minimum coverage to call a SNP]'
+```
+
+### `--var_fraction`
+
+Minimum variant fraction to call a SNP as float.
+
+```bash
+--var_fraction '[Minimum variant fraction to call a SNP]'
+```
+
+### `--conversions`
+
+Minimum number of T>C conversions in a read to call it a T>C read.
+
+```bash
+--conversions '[Number of conversions to call a T>C read]'
+```
+
+### `--baseQuality`
+
+Minimum base quality to call a T>C conversion as integer.
+
+```bash
+--baseQuality '[Minimum base quality for a T>C conversion]'
+```
+
+### `--readLength`
```

Comment:
Yeah, there are a bunch of these. Would be good if you could go through all params and make them snake case.

Reply:
NB: In the code later it looks like they may have already been changed? In which case, this documentation needs checking 👍

Reply:
Yes I made the changes in the code and forgot to update the usage.md.